usd_qtpy/editor.py

```diff
@@ -94,5 +102,23 @@ def update_panel_checkstate():
 
         panels_menu.aboutToShow.connect(update_panel_checkstate)
 
+        # Render menu
+
+        render_menu = menubar.addMenu("Render")
+        render_labels = "Playblast" , "Snapshot"
+        render_actions = {label : render_menu.addAction(label) for label in render_labels}
+
+        # Testing 
+        print("\n".join(render_util.getAllRenderEngineNames()))
+        goal_file = R"C:\dump\playblastview##.##.png"
+        
+        def render_snapshot(stage,stageview,path):
+            cam = render_util.playblast.cameraFromView(stage,stageview)
```

Comment:
I feel like if this is complex or requires a function to access the camera we should just expose it on the `stageview` instead:

```
cam = stageview.get_active_camera()
```

Reply:
This is a really weird function, from the stageview's end.

It basically puts a camera in the hierarchy of the scene so I am not at all sure what to do with all of that yet or where it should belong. It doesn't keep a reference to the current viewer camera, because the viewer camera isn't a USD cam, it's a "FreeCam" that basically wraps an openGL camera.

I have put it in the render_util because my rationale is, that maybe it'd be neat to have a seperate "camera from current view" button to use. I would need to set the internals of the camera to make sense for a default 16:9 camera though, as currently it will spawn a camera with a "sensor size" based on the widget size.

So, currently, I just don't quite know where to put it. All I do know is that I want the resulting camera from it,
so I can later hook it up to a turntable, for example.

Maybe this could benefit from a context wrapper, like
```py
with currentViewCam(stageView) as cam:
    do_stuff_wth_cam(cam)
```
and it'd delete the camera from the hierarchy later on.

I don't really know yet. We'll see.